wdl2cwl/tests/oop_cwl_files/minCores.cwl

```diff
@@ -0,0 +1,25 @@
+class: CommandLineTool
+id: Echo
+inputs: []
+outputs:
+  - id: out
+    type: stdout
+requirements:
+  - class: DockerRequirement
+    dockerPull: quay.io/biocontainers/bcftools:1.10.2--h4f4756c_2
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: example.sh
+        entry: |4
+
+            echo "Hello world"
+  - class: InlineJavascriptRequirement
+  - class: NetworkAccess
+    networkAccess: true
+  - class: ResourceRequirement
+    coresMin: $("8")
```

Comment:
```suggestion
    coresMin: 8
```

Reply:
This is wrong because `coresMIn` is of type `long`, or `float`; or a CWL Expression that returns a `long` or `float`; but not a string :-)